frontend/src/scenes/dashboard/dashboards/DashboardsTable.tsx

```diff
@@ -215,28 +216,31 @@ export function DashboardsTable({
                 />
                 <div className="flex items-center gap-4 flex-wrap">
                     <div className="flex items-center gap-2">
-                        <LemonButton
-                            active={filters.pinned}
-                            type="secondary"
-                            status="stealth"
-                            size="small"
-                            onClick={() => setFilters({ pinned: !filters.pinned })}
-                            icon={<IconPinOutline />}
-                        >
-                            Pinned
-                        </LemonButton>
-                    </div>
-                    <div className="flex items-center gap-2">
-                        <LemonButton
-                            active={filters.shared}
-                            type="secondary"
-                            status="stealth"
-                            size="small"
-                            onClick={() => setFilters({ shared: !filters.shared })}
-                            icon={<IconShare />}
-                        >
-                            Shared
-                        </LemonButton>
+                        <span>Filter to:</span>
```

Comment:
I agree that the "Filter to:" is useful UX-wise. This is a bit ugly because it's not totally clear visually that "Created by:" falls under "Filter to:", but I don't have a better solution right now. 
<img width="469" alt="Screenshot 2023-11-21 at 18 43 15" src="https://github.com/PostHog/posthog/assets/4550621/0f38ab3a-311d-44b4-9f0b-5d2ff630653d">